### PR TITLE
Add summary table entries for Prometheus

### PR DESCRIPTION
**Pull request by beorn7**
Signed-off-by: beorn7 <beorn@grafana.com>

_Pre-submission checklist removed because this PR only adds the new summary table entries to an already existing entry._

### CODE REVIEW

File: landscape.yml [hunk 1]
```diff
@@ -7012,6 +7012,13 @@ landscape:
               youtube_url: https://www.youtube.com/channel/UC4pLFely0-Odea4B2NL1nWA
               slack_url: https://freenode.net/
```
Comment (roidelapluie) [inline landscape.yml:7013]: This is incorrect

Comment (beorn7) [reply landscape.yml:7013]: Yes, needs to be updated to `https://libera.chat/`. @roidelapluie are you already working on a PR?

Comment (beorn7) [reply landscape.yml:7013]: Apparently not. I'll send one in a minute.

Comment (beorn7) [reply landscape.yml:7013]: https://github.com/cncf/landscape/pull/2899

**Pull request merged**